echopype/clean/api.py

```diff
@@ -185,14 +278,16 @@ def mask_attenuated_signal(
     ----------
     ds_Sv : xarray.Dataset
         Calibrated Sv data with depth data variable.
-    upper_limit_sl : Union[int, float], default `400`m
+    upper_limit_sl : str, default `400m`
         Upper limit of deep scattering layer line (m).
-    lower_limit_sl : Union[int, float], default `500`m
+    lower_limit_sl : str, default `500m`
         Lower limit of deep scattering layer line (m).
     num_side_pings : int, default `15`
         Number of preceding & subsequent pings defining the block.
-    attenuation_signal_threshold : Union[int, float], default `8.0`dB
+    attenuation_signal_threshold : str, default `8.0dB`
```

Comment:
Could you change ` ` here to " " like elsewhere?

Reply:
Made this change 👍